client/dashboard/sites/logs/index.tsx

```diff
@@ -168,6 +195,15 @@ function SiteLogs( { logType }: { logType: LogType } ) {
 
 	return (
 		<PageLayout header={ <PageHeader title={ __( 'Logs' ) } /> }>
+			<DateRangePicker
```

Comment:
I think it would be better to move it into the main props of the `CalloutOverlay`